config/manifests/bases/piraeus-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: piraeus-operator.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: LinstorCluster is the Schema for the linstorclusters API
+      displayName: Linstor Cluster
+      kind: LinstorCluster
+      name: linstorclusters.piraeus.io
+      version: v1
+    - description: LinstorSatelliteConfiguration is the Schema for the linstorsatelliteconfigurations
+        API
+      displayName: Linstor Satellite Configuration
+      kind: LinstorSatelliteConfiguration
+      name: linstorsatelliteconfigurations.piraeus.io
+      version: v1
+    - description: LinstorSatellite is the Schema for the linstorsatellites API
+      displayName: Linstor Satellite
+      kind: LinstorSatellite
+      name: linstorsatellites.piraeus.io
+      version: v1
+  description: Some operator
+  displayName: piraeus-operator
+  icon:
+  - base64data: ""
+    mediatype: ""
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - storage
+  links:
+  - name: Piraeus Operator
+    url: https://piraeus-operator.domain
+  maturity: alpha
+  provider:
+    name: Piraeus Datastore
+    url: https://piraeus.io
+  version: 0.0.0
```

Comment:
Is this overridden somewhere?

Reply:
It gets overridden when running `make bundle`